<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpreadsheetDock</class>
 <widget class="QWidget" name="SpreadsheetDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>694</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="14" column="0" colspan="2">
    <widget class="QLabel" name="lLinkingEnabled">
     <property name="text">
      <string>Enabled</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="ResizableTextEdit" name="teComment">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QCheckBox" name="cbShowSparklines">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="3">
    <widget class="QSpinBox" name="sbColumnCount">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>16777215</number>
     </property>
    </widget>
   </item>
   <item row="14" column="3">
    <widget class="QCheckBox" name="cbLinkingEnabled">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lName">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>58</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="15" column="3">
    <widget class="TreeViewComboBox" name="cbLinkedSpreadsheet"/>
   </item>
   <item row="0" column="3">
    <widget class="TimedLineEdit" name="leName"/>
   </item>
   <item row="15" column="0" colspan="2">
    <widget class="QLabel" name="lLinkedSpreadsheet">
     <property name="text">
      <string>Spreadsheet</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>58</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="lColumnCount">
     <property name="text">
      <string>Columns:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="lDimensions">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Dimensions</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QSpinBox" name="sbRowCount">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="lShowSparklines">
     <property name="text">
      <string>Sparklines:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lComment">
     <property name="text">
      <string>Comment:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="lFormat">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Column Header</string>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QCheckBox" name="cbShowComments">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="lRowCount">
     <property name="text">
      <string>Rows:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QLabel" name="lShowComments">
     <property name="text">
      <string>Comments:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QLabel" name="lLinking">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Linking</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ResizableTextEdit</class>
   <extends>QTextEdit</extends>
   <header>frontend/widgets/ResizableTextEdit.h</header>
  </customwidget>
  <customwidget>
   <class>TimedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>frontend/widgets/TimedLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>TreeViewComboBox</class>
   <extends>QComboBox</extends>
   <header>TreeViewComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leName</tabstop>
  <tabstop>teComment</tabstop>
  <tabstop>sbRowCount</tabstop>
  <tabstop>sbColumnCount</tabstop>
  <tabstop>cbShowComments</tabstop>
  <tabstop>cbShowSparklines</tabstop>
  <tabstop>cbLinkingEnabled</tabstop>
  <tabstop>cbLinkedSpreadsheet</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
